addons/crm_project_issue/__openerp__.py

```diff
@@ -12,8 +13,8 @@
 Link module to map leads to issues
         """,
     'data': [
-        'project_issue_view.xml'
+        'wizard/crm_lead2projectissue_wizard_view.xml',
+        'views/crm_lead_views.xml'
```

Comment:
please arrange alphabetically like..
```
A for apple
B for Ball or Bird
```
:laughing: 
![alphabet-coloring-pages-printable-apple-and-bird](https://cloud.githubusercontent.com/assets/14940179/12080145/29cf621a-b20f-11e5-8902-ce583c0612cc.gif)